tools/libSwiftSyntaxParser/CMakeLists.txt

```diff
@@ -45,16 +55,25 @@ endif()
 
 set_property(TARGET libSwiftSyntaxParser APPEND_STRING PROPERTY
   COMPILE_FLAGS " -fblocks")
+set_property(TARGET libSwiftSyntaxParserStatic APPEND_STRING PROPERTY
+  COMPILE_FLAGS " -fblocks")
 if(NOT CMAKE_SYSTEM_NAME STREQUAL Darwin)
   target_link_libraries(libSwiftSyntaxParser PRIVATE
     BlocksRuntime)
+  target_link_libraries(libSwiftSyntaxParserStatic PRIVATE
+    BlocksRuntime)
 endif()
 
 add_dependencies(parser-lib libSwiftSyntaxParser)
+add_dependencies(parser-lib libSwiftSyntaxParserStatic)
 swift_install_in_component(TARGETS libSwiftSyntaxParser
   ARCHIVE DESTINATION "lib${LLVM_LIBDIR_SUFFIX}/swift/${SWIFT_SDK_${SWIFT_HOST_VARIANT_SDK}_LIB_SUBDIR}" COMPONENT parser-lib
   LIBRARY DESTINATION "lib${LLVM_LIBDIR_SUFFIX}/swift/${SWIFT_SDK_${SWIFT_HOST_VARIANT_SDK}_LIB_SUBDIR}" COMPONENT parser-lib
   RUNTIME DESTINATION "bin" COMPONENT parser-lib)
+swift_install_in_component(TARGETS libSwiftSyntaxParserStatic
+  ARCHIVE DESTINATION "lib${LLVM_LIBDIR_SUFFIX}/swift/${SWIFT_SDK_${SWIFT_HOST_VARIANT_SDK}_LIB_SUBDIR}" COMPONENT parser-lib
+  LIBRARY DESTINATION "lib${LLVM_LIBDIR_SUFFIX}/swift/${SWIFT_SDK_${SWIFT_HOST_VARIANT_SDK}_LIB_SUBDIR}" COMPONENT parser-lib
```

Comment:
I'm hoping this translates to being included in produced toolchains so it could be shipped with releases, but I'm not sure if I should build one of those to see, or if it needs to also be added below